client/client.go

```diff
@@ -350,64 +410,16 @@ func newHttpsClient(conf Config) (*http.Client, error) {
 
 // Client configuration
 
-type Config struct {
-	ServerCert string
-	IsHttps    bool
-	NoVerify   bool
```

Comment:
I don't understand, are we removing the option to skip CA validation entirely?

Reply:
Perhaps you are right, I removed it because this was set from within the `HttpsClient` struct. 
I guess I can add a new configuration option directly, to set this